code/modules/paperwork/handlabeler.dm

```diff
@@ -83,11 +83,9 @@
 /obj/item/weapon/hand_labeler/attackby(obj/item/I, mob/user, params)
 	..()
 	if(istype(I, /obj/item/hand_labeler_refill))
-		if(!user.unEquip(I))
-			return
 		user << "<span class='notice'>You insert [I] into [src].</span>"
 		qdel(I)
-		labels_left = initial(labels_left)
+		labels_left += initial(labels_left)

```

Comment:
this lets you get more than ```initial(labels_left)``` labels, which Is not the intention.
for example the base has 30, so it's capped at a roll of 30.
yes it wastes the refill to use it at any less, but it's still a cap of 30, you've just broken the cap.